docs/api/admin_users.rst

```diff
@@ -0,0 +1,584 @@
+                "plainPassword": "youShallNotPass",
+                "localeCode": "en_US"
+            }
+        '
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 201 CREATED
+
+.. code-block:: json
+
+    {
+        "id": 7,
+        "username": "Barlog",
+        "usernameCanonical": "barlog",
+        "roles": [
+            "ROLE_ADMINISTRATION_ACCESS"
+        ],
+        "email": "teamEvil@middleearth.com",
+        "emailCanonical": "teamevil@middleearth.com",
+        "enabled": false
+    }
+
+.. warning::
+
+    If you try to create an admin user without username, email, password or locale's code , you will receive a ``400 Bad Request`` error, that will contain validation errors.
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/users/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+       -X POST
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 400 Bad Request
+
+.. code-block:: json
+
+    {
+        "code": 400,
+        "message": "Validation Failed",
+        "errors": {
+            "children": {
+                "username": {
+                    "errors": [
+                        "Please enter your name."
+                    ]
+                },
+                "email": {
+                    "errors": [
+                        "Please enter your email."
+                    ]
+                },
+                "plainPassword": {
+                    "errors": [
+                        "Please enter your password."
+                    ]
+                },
+                "enabled": {},
+                "firstName": {},
+                "lastName": {},
+                "localeCode": {
+                    "errors": [
+                        "Please choose a locale."
+                    ]
+                }
+            }
+        }
+    }
+
+You can also create an admin user with additional (not required) fields:
+
++-----------+----------------+---------------------------------+
+| Parameter | Parameter type | Description                     |
++===========+================+=================================+
+| enabled   | request        | Flag set if the user is enabled |
++-----------+----------------+---------------------------------+
+| firstName | request        | The admin user's first name     |
++-----------+----------------+---------------------------------+
+| lastName  | request        | The admin user's last name      |
++-----------+----------------+---------------------------------+
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/users/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X POST \
+        --data '
+            {
+                "firstName": "Barlog",
+                "lastName": "Barlog!",
```

Comment:
I would vote for the `of Morgoth` as the last name, as this creature is widely known as **Balrog of Morgoth** ;)